.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
```

Comment:
doesn't need to be labeled [Feature] because that's the purpose of tags in GitHub issues/PRs

Reply:
This target a commit that should not be here